metadata.rb

```diff
@@ -7,7 +7,7 @@
 source_url 'https://github.com/michaelklishin/cassandra-chef-cookbook' if respond_to?(:source_url)
 issues_url 'https://github.com/michaelklishin/cassandra-chef-cookbook/issues' if respond_to?(:issues_url)
 version '4.6.0'
-depends 'java'
+depends 'java', '< 8.0.0'
```

Comment:
@ankit-agarwal-E1304 As of now, we have java 8 installed on the machine. Will this install a lower version of java on Cassandra machine?

Reply:
requires JDK8
[Ref](https://github.com/michaelklishin/cassandra-chef-cookbook#support-jdk-versions)